### DISCUSSION TITLE
请问能否给出参考资料

### DISCUSSION THREAD

**Discussion by Saulwonderfulman** [SylarLong/iztro#177] [General]
hello~

1.在平板上排盘时文字显示会乱掉,如图所示.(设备:小米平板6pro,浏览器为雨见浏览器)
![fig](https://github.com/SylarLong/iztro/assets/67410761/c70eda2a-e207-4fb9-925e-7c4cca4c7477)

2.文档看下来感觉很好,能否在文档最后一些参考资料,仅仅列出学斗数需要看的文献就可以,这将会对入门者帮助很大同时又不会增加很大的工作量

**Comment by SylarLong** [comment]
感谢提供截图以供改进，目前犹豫内容过多的原因所以会将界面撑开变形，后续会修复。由于参考资料比较繁杂，我是经过多方资料加上自己的验证归纳总结出来的文档，所以一个结论会涉及到不通的信息来源，很多信息来源我自己都忘记出处了，所以无法准确给出参考文献。感谢你的支持与关注～

**Reply by Saulwonderfulman** [reply]
感谢回复,表示理解,祝作者生活愉快~ : )
